tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -229,3 +303,94 @@ def test_should_respect_page_size_limit_default(self, test_client):
 
         assert response.status_code == 200
         assert len(response.json()["assets"]) == 100
+
+
+class TestGetAssetsEvents(TestAssets):
```

Comment:
Comparing with test_asset_endpoint.py from legacy API, can we try to maintain similar functions? 

I think these are missing/renamed:
1. `test_filtering` -> has general filtering compared to `test_filter_events_by_asset_id`
2. `test_includes_created_dagrun`

Reply:
Ok great makes sense.